app/src/layouts/appLayout/sidebar/sidebar.jsx

```diff
@@ -1,24 +1,95 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-ui
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { Component } from 'react';
 import classNames from 'classnames/bind';
-import { NavLink } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+import { SidebarButton } from 'components/buttons/sidebarButton/sidebarButton';
+import PropTypes from 'prop-types';
 import styles from './sidebar.scss';
+import DashboardIcon from './img/dashboard-icon-inline.svg';
+import LaunchesIcon from './img/launches-icon-inline.svg';
+import FiltersIcon from './img/filters-icon-inline.svg';
+import DebugIcon from './img/debug-icon-inline.svg';
+import ProfileIcon from './img/profile-icon-inline.svg';
+import AdministrateIcon from './img/administrate-icon-inline.svg';
+import LogoutIcon from './img/logout-icon-inline.svg';
 
 const cx = classNames.bind(styles);
 
-const SidebarItem = ({ link, name }) => (
-  <NavLink to={link} className={cx('item')} activeClassName={cx('active')}>{name}</NavLink>
-);
-SidebarItem.propTypes = {
-  link: PropTypes.string.isRequired,
-  name: PropTypes.string.isRequired,
-};
+export class Sidebar extends Component {
+
+  static propTypes = {
+    onClickNavBtn: PropTypes.func,
```

Comment:
I think it's just `onClick` handler since it is used on the component root element.

Reply:
fixed